script/fake_data.py

```diff
@@ -10,50 +10,86 @@
 """
 
 import os
-from datetime import datetime, timezone
+import sys
+from datetime import UTC, datetime, timezone
 
 from nowcasting_datamodel.connection import DatabaseConnection
-from nowcasting_datamodel.fake import make_fake_forecasts, make_fake_gsp_yields
+from nowcasting_datamodel.fake import (
+    N_FAKE_FORECASTS,
+    generate_fake_forecasts,
+    make_fake_gsp_yields,
+)
 from nowcasting_datamodel.models.forecast import (
     ForecastSQL,
     ForecastValueLatestSQL,
+    ForecastValueSevenDaysSQL,
     ForecastValueSQL,
 )
 from nowcasting_datamodel.models.models import StatusSQL
+from nowcasting_datamodel.save.save import save as save_forecasts
+from sqlalchemy import inspect
+
+from nowcasting_api.utils import floor_30_minutes_dt
 
-from src.utils import floor_30_minutes_dt
+sys.path.append("/app/nowcasting_api")
```

Comment:
could you add a comment, becasue I think this only works in the docker file, not running locally

Reply:
in local there is no such thing as `app` this is only introduced when running the api in docker